Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN apk update && apk --no-cache --quiet add --update \
     py3-pip
 
 RUN pip3 install --upgrade --no-cache-dir pip \
-    && pip3 install --upgrade --no-cache-dir hokusai
+    && pip3 install --upgrade --no-cache-dir hokusai --ignore-installed
 
```

Comment:
Had to add the flag because the CI built failed with a preinstalled packaging version